site/pt/_partials/devtools/whats-new.md

```diff
@@ -5,28 +5,27 @@ Consulte a versão em inglês do <a href="/tags/new-in-devtools/" translate="no"
 {# $content #}
 
 
-{# ### Chrome 111 {: #chrome111 }
+### Chrome 111 {: #chrome111 }
 
-* [Debugging HD color with the Styles pane](/pt/blog/new-in-devtools-111/#color)
-* [Enhanced breakpoint UX](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
-* [Customizable Recorder shortcuts](/pt/blog/new-in-devtools-111/#recorder)
-* [Better syntax highlight for Angular](/pt/blog/new-in-devtools-111/#syntax)
-* [Reorganize caches in the Application panel](/pt/blog/new-in-devtools-111/#cache)
-* [Miscellaneous highlights](/pt/blog/new-in-devtools-111/#misc)
- #}
+* [Debugging de cores HD no painel de Estilos](/pt/blog/new-in-devtools-111/#color)
+* [Melhorias na UX dos breakpoints](/pt/blog/new-in-devtools-111/#breakpoint-redesign)
+* [Atalhos customizáveis no Gravador](/pt/blog/new-in-devtools-111/#recorder)
+* [Melhor syntax highlighting para Angular](/pt/blog/new-in-devtools-111/#syntax)
```

Comment:
Similar to the above, we've used "realce de sintaxe" in previous articles ([example](https://github.com/GoogleChrome/developer.chrome.com/blob/10bdcea52ff4c78a301d5e56327f16a04fafa2e6/site/pt/blog/new-in-devtools-109/index.md?plain=1#L130))